portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,29 +27,45 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+//An object that will be converted to JSON. Contains comments and their sentiment scores
+class CommentSentiment {
+    ArrayList<String> comments;
+    ArrayList<Double> scores;
+
+    CommentSentiment(ArrayList<String> c, ArrayList<Double> s) {
+        this.comments = c;
+        this.scores = s;
+    }
+}
+
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
     //Loads comments every time page is loaded
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         response.setContentType("application/json");
-        ArrayList<String> comments = new ArrayList<>();
+
+        CommentSentiment commentSentiment = 
+            new CommentSentiment(new ArrayList<String>(), new ArrayList<Double>());
         
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
         PreparedQuery results = datastore.prepare(query);
 
         for (Entity entity : results.asIterable()) {
             String comment = (String) entity.getProperty("comment");
-            comments.add(comment);
+            Double score = (Double) entity.getProperty("sentiment");
+            commentSentiment.comments.add(comment);
+            commentSentiment.scores.add(score);
         }
 
-
-        Gson gson = new Gson();
-        String jsonComments = gson.toJson(comments);
+        String jsonComments = new Gson().toJson(commentSentiment);
```

Comment:
really cool idea - keeps converting from Java objects to a string really simple :)